### PR TITLE
Fixing new POET bugs, and old bugs with multwhite fits, masked values in fits, nbin_plot in S5, and S6 model loading

### PR DESCRIPTION
**Pull request by taylorbell57**
Fixing new POET bugs and old bugs

Old bugs include issues with multwhite fits, masked values, and nbin_plot dealing with nbin_plot=len(time) in the presence of masked values.

New POET bugs include improperly handling multi-channel (or multwhite) fits (because only the values for channel 0 were ever used), and a broken alias between fp and fpfs.

Also fixing a bug in S6 related to load_model function.

### CODE REVIEW

**Comment by taylorbell57** [comment]
This PR includes patches that will be merged into PR #609, and this PR should be merged first

File: src/eureka/S5_lightcurve_fitting/models/GPModel.py [hunk 1]
```diff
@@ -164,14 +164,9 @@ def eval(self, fit, channel=None, gp=None, **kwargs):
                 unc_fit = self.unc_fit
             residuals = flux-fit
 
-            time = self.time
-            if self.multwhite:
-                # Split the arrays that have lengths of the original time axis
-                time = split([time, ], self.nints, chan)[0]
-
```
[the remark this answers is not in the document]
Comment (taylorbell57) [reply src/eureka/S5_lightcurve_fitting/models/GPModel.py:171]: Yeah, we just didn't use time so there was no need to do this

File: src/eureka/S5_lightcurve_fitting/models/PoetModels.py [hunk 2]
```diff
@@ -394,7 +409,7 @@ def trnlldsp(z, rprs, u):
                 - u3 * (1. - 4. / 7. * np.sqrt(np.sqrt(x * x * x))) \
                 - u4 * (1. - 4. / 8. * x)
     y[iingress] = 1. - I1star \
-        * (rprs ** 2 * np.arccos((z[iingress] - 1.) / rprs) - (z[iingress] - 1.)
+        * (rprs**2 * np.arccos((z[iingress] - 1.) / rprs) - (z[iingress] - 1.)
            * np.sqrt(rprs ** 2 - (z[iingress] - 1.) ** 2)) / np.pi / Sigma4
```
[the remark this answers is not in the document]
Comment (taylorbell57) [reply src/eureka/S5_lightcurve_fitting/models/PoetModels.py:413]: Can do - I just did it for that one line for flake8 compliance (line was too long)

File: src/eureka/S5_lightcurve_fitting/plots_s5.py [hunk 3]
```diff
@@ -382,15 +388,13 @@ def plot_rms(lc, model, meta, fitter):
                 label='Gaussian Std. Err.', zorder=1)
 
         # Format the main axes
-        ax.set_xlim(0.95, binsz[-1] * 2)
-        ax.set_ylim(stderr[-1] / normfactor / 2., stderr[0] / normfactor * 2.)
```
[the remark this answers is not in the document]
Comment (taylorbell57) [reply src/eureka/S5_lightcurve_fitting/plots_s5.py:386]: As it was, it was a bit tighter on the left, but less tight on the right and was cropping off the bottom in some cases where the black line falls below the red line near the right end. The default axis limits looked a fair bit nicer to me after experimenting

**Comment by taylorbell57** [comment]
@kevin218 alright, I believe I've fixed the key remaining bugs. The one remaining question I have is the intended units for the `cos1_amp` terms. At present, a value of 0.5 gives a nightside flux (aka at mid-transit) of zero - that's how the `SinusoidPhaseCurve` terms work and is totally fine, but you or others may desire for a value of 1 to mean zero nightside flux (as I assume you intend based on your template EPFs). If you do want this, then you can just multiply the values by 0.5 within the POET code to allow `cos1_amp` to vary between zero and one.

**Comment by kevin218** [comment]
@taylorbell57 I resolved the issue with the phase curve amplitude in #609.  It was a little more complicated than either of us probably expected, but I'm happy with the final result.

**Pull request merged**